toolkit/tools/pkg/imagecustomizerlib/customizepartitions_test.go

```diff
@@ -18,25 +18,35 @@ import (
 )
 
 func TestCustomizeImagePartitions(t *testing.T) {
-	baseImage := checkSkipForCustomizeImage(t, baseImageTypeCoreEfi)
-	testCustomizeImagePartitionsToEfi(t, "TestCustomizeImagePartitions", baseImage)
+	for _, version := range supportedAzureLinuxVersions {
+		t.Run(string(version), func(t *testing.T) {
+			testCustomizeImagePartitionsToEfi(t, "TestCustomizeImagePartitions"+string(version),
+				baseImageTypeCoreEfi, version)
+		})
+	}
 }
 
 func TestCustomizeImagePartitionsLegacyToEfi(t *testing.T) {
-	baseImage := checkSkipForCustomizeImage(t, baseImageTypeCoreLegacy)
-	testCustomizeImagePartitionsToEfi(t, "TestCustomizeImagePartitionsLegacyToEfi", baseImage)
+	for _, version := range supportedAzureLinuxVersions {
+		t.Run(string(version), func(t *testing.T) {
+			testCustomizeImagePartitionsToEfi(t, "TestCustomizeImagePartitionsLegacyToEfi"+string(version),
+				baseImageTypeCoreLegacy, version)
+		})
+	}
 }
 
-func testCustomizeImagePartitionsToEfi(t *testing.T, testName string, baseImage string) {
-	var err error
+func testCustomizeImagePartitionsToEfi(t *testing.T, testName string, imageType baseImageType,
+	imageVersion baseImageVersion,
+) {
+	baseImage := checkSkipForCustomizeImage(t, imageType, imageVersion)
 
 	testTmpDir := filepath.Join(tmpDir, testName)
 	buildDir := filepath.Join(testTmpDir, "build")
 	configFile := filepath.Join(testDir, "partitions-config.yaml")
 	outImageFilePath := filepath.Join(testTmpDir, "image.raw")
 
 	// Customize image.
-	err = CustomizeImageWithConfigFile(buildDir, configFile, baseImage, nil, outImageFilePath, "raw", "",
+	err := CustomizeImageWithConfigFile(buildDir, configFile, baseImage, nil, outImageFilePath, "raw", "",
```

Comment:
Why not just initialize the err variable? Is this a code style choice, or do you recommend initializing it with `:=`?

Reply:
Either way works. But using `:=` reduces the number of lines of code.